MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -46,23 +46,35 @@
+                             resolve_split_vertices_tight = cms.bool(True), # merging routine, based on vtx dphi and dVV 
+                             investigate_merged_vertices = cms.bool(False), # investigate quality cuts on merged vertices from tight merging 
+                             resolve_shared_jets = cms.bool(True),       # shared-jet mitigation
+                             resolve_shared_jets_src = cms.InputTag('selectedPatJets'), 
                              beamspot_src = cms.InputTag('offlineBeamSpot'),
                              seed_tracks_src = cms.InputTag('mfvVertexTracks', 'seed'),
                              n_tracks_per_seed_vertex = cms.int32(2),
                              max_seed_vertex_chi2 = cms.double(5),
                              use_2d_vertex_dist = cms.bool(False),
                              use_2d_track_dist = cms.bool(False),
                              merge_anyway_dist = cms.double(-1),
-                             merge_anyway_sig = cms.double(3),
+                             merge_anyway_sig = cms.double(4), # merging criteria for loose merging (*only* if resolve_split_vertices_loose is True)
                              merge_shared_dist = cms.double(-1),
-                             merge_shared_sig = cms.double(4),
+                             merge_shared_sig = cms.double(4), # default merging shared-track vertices  
                              max_track_vertex_dist = cms.double(-1),
-                             max_track_vertex_sig = cms.double(5),
-                             min_track_vertex_sig_to_remove = cms.double(1.5),
+                             max_track_vertex_sig = cms.double(5), # default track arbitration
+                             min_track_vertex_sig_to_remove = cms.double(1.5), # default track arbitration
                              remove_one_track_at_a_time = cms.bool(True),
                              max_nm1_refit_dist3 = cms.double(-1),
                              max_nm1_refit_distz = cms.double(0.005),
                              max_nm1_refit_count = cms.int32(-1),
+                             trackrefine_sigmacut = cms.double(5), # track refinement criteria (*only* if do_track_refinement = True)
+                             trackrefine_trimmax = cms.double(5), # track refinement criteria (*only* if do_track_refinement = True)
                              histos = cms.untracked.bool(True),
+                             histos_noshare = cms.untracked.bool(True),   # make plots of no shared-track vertices 
+                             histos_output_afterdzfit = cms.untracked.bool(True),   # make plots of output vertices after the default vertexing 
+                             histos_output_aftermerge = cms.untracked.bool(True),   # make plots of output vertices after the default vertexing  + tight merging routine turned on
```

Comment:
I forget, did we plan to turn these off by default?

Reply:
Maybe it was only the afterdzfit and aftermerge?

Reply:
If you mean the final histos with dzfit + merge + sharedjets, then we should only turn on aftersharedjets.

Reply:
Done.